src/main/docs/guide/gettingStarted.adoc

```diff
@@ -2,13 +2,28 @@ To get started add Micronaut's `openapi` to the annotation processor scope of yo
 
 dependency:micronaut-openapi[scope="annotationProcessor", groupId="io.micronaut.openapi"]
 
-NOTE: For Kotlin the `openapi` dependency should be in the `kapt` scope and for Groovy in the `compileOnly` scope.
+[NOTE]
+====
+For Kotlin the `openapi` dependency should be in the `kapt` or `ksp` scope and for Groovy in the `compileOnly` scope.
+====
+
+[IMPORTANT]
+====
+If you use KSP need to disable incremental processing:
+[properties]
+----
+ksp.incremental=false
```

Comment:
why do you need to disable incremental processing for KSP?

Reply:
@sdelamo Look to this: https://github.com/micronaut-projects/micronaut-openapi/issues/1776 and this: https://github.com/micronaut-projects/micronaut-core/issues/11215, please.

After this issue will be fixed - we can remove this note. But until then this is the only way to work correctly with KSP.